prov/opx/include/rdma/opx/fi_opx_endpoint.h

```diff
@@ -1409,6 +1512,17 @@ void complete_receive_operation_internal (struct fid_ep *ep,
 			if (OFI_LIKELY(niov == 1)) {
 				assert(payload != NULL);
 
+				uint64_t rbuf_device;
+				enum fi_hmem_iface rbuf_iface;
```

Comment:
I understand the existing code in the opx provider has already broken the rule, but the convention is not to interleave declarations with statements. I will leave your team to decide if a refactoring is needed in the future.